doc/classes/PopupMenu.xml

```diff
@@ -91,6 +107,8 @@
 			<argument index="3" name="global" type="bool" default="false">
 			</argument>
 			<description>
+				Adds an icon and assigns the specified [ShortCut] and [Texture] to it.
```

Comment:
No need to add links to the type of the arguments: they're all clickable links in the docs already. You can use [code][/code] instead to highlight the arguments:

`Adds an icon and assigns the [code]shortcut[/code] and [code]texture[/code] to it.`

Reply:
May i ask what is [code][/code] tag for? Or at least what it will be for since right now it seems to do nothing.

Reply:
Also, i though there is a good reason that you can enclose them. Making them as links seems to simplify them (unless code tags will add extra functionality).

Reply:
Plus @akien-mga seems to be using the links alot (from which i'm taking examples). And it makes sense, it adds convenience while it does not harm.

Reply:
Code tags are supposed to produce `this kind of result`: highlight inline code without creating a link. I don't mind either that or links, I'm not sure if there's a clear guideline in this case.